src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -203,9 +209,13 @@ function ReportActionItemMessageEdit(props) {
             const {text: newDraft, emojis} = EmojiUtils.replaceAndExtractEmojis(newDraftInput, props.preferredSkinTone, props.preferredLocale);
 
             if (!_.isEmpty(emojis)) {
-                insertedEmojis.current = [...insertedEmojis.current, ...emojis];
-                debouncedUpdateFrequentlyUsedEmojis();
+                const newEmojis = EmojiUtils.getAddedEmojis(emojis, emojisPresentBefore.current);
```

Comment:
Can we move this block to a util or an action? I can see the exact same thing repeated for `ComposeWithSuggestions`

Reply:
I am not sure how to do this cleanly with the debounce, also it was repeating before and a lot of code is repeating in those two files, can we keep it like this for now ?